CMSgov/mint-app
# [NOREF] Updated GA4 custom dimensions

# NOREF

## Description


 - Added job codes - `user_group` user property to GA data stream
 - Added email domain - `domain` user property to GA data stream


## How to test this change

1.  Verify `domain` and `user_group` metrics appear on realtime dashboard
2. In the User Segment Explorer, verify the `user_id` hash is set (not the previous LD userKey)

![Screenshot 2024-08-19 at 1 32 39 PM](https://github.com/user-attachments/assets/f3808149-298d-41e7-8316-48967192a519)


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
